ota-plus-web/app/com/advancedtelematic/controllers/ClientToolController.scala

```diff
@@ -156,7 +156,9 @@ class ClientToolController @Inject()(
         }
       }
       val authParams = if(conf.getOptional[String]("authplus.host").isDefined) {
-        await(oidcGateway.getUserInfo(accessToken).flatMap(ic => getOAuth2Params(namespace, ic.userId)))
+        // TODO OTA-2846 AuthPlus should validate this userId in the token; currently it doesn't
```

Comment:
I'm not sure anymore that this comment is accurate. Auth-plus validates token based on signature, and it sets the subject of that token by client-id. We need to revisit how we use these tokens here, or wait until auth-plus is removed altogether.

Reply:
Should I just rephrase the comment then or do we want to make more changes in this PR?

Reply:
Just rephrase comment.